src/assets/js/csv.js

```diff
@@ -0,0 +1,19 @@
+
+function handleCsv(csv) {
+    const lines = csv.split("\r\n");
+    const res = [];
+    lines.splice(0, 1);
+    for (let line of lines) {
+        let elements = line.split(";");
+        let date = elements[0].split("-");
+        res.push({
+            date: new Date(parseInt(date[0]), parseInt(date[1]) - 1, parseInt(date[2]) + 1),
+            description: elements[1],
+            accountName: elements[4],
+            amount: parseFloat(elements[3])
+        })
+    }
+    return res;
+}
```

Comment:
Ik weet niet wat hier allemaal gebeurd met die indexes. Je kan ook kijken naar iets als een csv reader package. Dan weetje zeker dat alles werkt. En anders zou ik het zo doen. 

```suggestion
export function handleCsv(csv, delimiter = ';') {
    const rows = csv.split("\r\n");
    rows.splice(0, 1);

    return rows.map(row=> {
        const elements = row.split(delimiter)
        const date = elements[0].split('-')
        return {
            date: new Date(parseInt(date[0]), parseInt(date[1]) - 1, parseInt(date[2]) + 1),
            description: elements[1],
            accountName: elements[4],
            amount: parseFloat(elements[3])
        }
    })
}
```